src/vl-map-vector-layer.js

```diff
@@ -54,6 +60,7 @@ export class VlMapVectorLayer extends VlMapLayer {
       this._styles = [];
       this._layer.setStyle(style);
     }
+    this.dispatchEvent(new CustomEvent(VlMapVectorLayer.EVENTS.styleChanged, {bubbles: true, composed: true, detail: {style: style}}));
```

Comment:
Ik denk dat bubbels default true is, maar geen probleem om dat expliciet nog eens te vermelden.